README.md

```diff
@@ -1,112 +1,142 @@
 ```
 
 Use in a playbook:
 
-```
+```yml
 ---
 - hosts: someserver
   roles:
     - { role: airflow }
 ```
 
-## Usage
+## Usage 🏃
+
+Look to the defaults properties files to see the possible configuration properties, take a look for them:
 
-Look to the defaults properties file to see the possible configuration properties.
+- [`main.yml`](./defaults/main/main.yml) for airflow general purpose
+- [`airflow-cfg.yml`](./defaults/main/airflow-cfg.yml) for all the related airflow.cfg config parameters
+- [`webserver-config-py.yml`](./defaults/main/webserver-config-py.yml) for all the related webserver_config.py config parameters
 
-Bear in mind that, starting with Airflow v1.10.0, PyPi package `pyasn1` v0.4.4 is needed. To install it:
-``` yml
+👉 Don't forget :
+
+- 🔑 To set Fernet key.
+- 🔑 To set webserver secret key.
+- 📝 To set your AIRFLOW_HOME and AIRFLOW_CONFIG at your own discretion.
+- 📝 To set your installation and config skelton paths at your own discretion.
+  - 👉 See `airflow_skeleton_paths` in [`main.yml`](./defaults/main/main.yml)
+- 🐍 Python and pip version.
+- 📦 [Extra packages](#Extra-packages) if you need additional operators, hooks, sensors...
+- 📦 [Required Python packages](#Required-Python-packages) with version specific like SQLAlchemy for example (to avoid known Airflow bugs❗️) like below or because are necessary
+- ⚠️ With Airflow v1.10.0, PyPi package `pyasn1` v0.4.4 is needed. See examples below
+
+### 📦 Required Python packages
+
+[`airflow_required_python_packages`](./defaults/main/main.yml) should be a list following this format:
+
+```yml
 airflow_required_python_packages:
+  - { name: SQLAlchemy, version: 1.3.23 }
+  - { name: psycopg2 }
   - {name: pyasn1, version: 0.4.4}
 ```
 
-`airflow_extra_packages` (available at: https://airflow.apache.org/installation.html#extra-packages) should be a list following this format:
-``` yml
+### 📦 Extra packages
+
+[`airflow_extra_packages`](./defaults/main/main.yml) should be a list following this format:
+
+```yml
 airflow_extra_packages:
+  - apache.atlas
   - celery
-  - mysql
+  - ssh
 ```
-## Testing
 
-```
-pipenv install -r test-requirements.txt --python 3.8
+👉 For more info about this extra packages see: [Airflow extra packages](https://airflow.apache.org/docs/apache-airflow/stable/extra-packages-ref.html)
+
+## Testing 🧪
+
+```shell
+pipenv install -r test-requirements.txt --python 3.7
 pipenv run molecule test
 ```
 
-## Built With
+## Built With 🏗️
 
-![Ansible](https://img.shields.io/badge/ansible-2.4.5.0-green.svg)
+![Ansible](https://img.shields.io/badge/ansible-2.9.9-green.svg)
 
-## Versioning
+## Versioning 🗂️
 
 For the versions available, see the [tags on this repository](https://github.com/idealista/airflow-role/tags).
 
-Additionaly you can see what change in each version in the [CHANGELOG.md](CHANGELOG.md) file.
+Additionally you can see what change in each version in the [CHANGELOG.md](CHANGELOG.md) file.
```

Comment:
Maybe it's a good opportunity to [create an issue](https://github.com/idealista/cookiecutter-ansible-role/issues) to fix this typo :)

Reply:
Done!